projects/angular-ui/src/lib/snack-bar/snack-bar-container.ts

```diff
@@ -367,7 +367,12 @@ export class BaoSnackBarContainerComponent
             }
 
             inertElement.removeAttribute('aria-hidden');
-            liveElement.appendChild(inertElement);
+            try {
+              liveElement.appendChild(inertElement);
```

Comment:
this code throws an error in Angular 15 (see storybook (the live version at https://services.montreal.ca/bao-storybook/?path=/story/components-alert--dismissable-alert))

Reply:
https://github.com/VilledeMontreal/angular-ui/issues/193